scripts/insert_snapshot_warning.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+const fs = require('fs');
+const { JSDOM } = require('jsdom');
+const { execSync } = require('child_process');
+
+const COMMIT = execSync('git rev-parse --verify HEAD').toString();
+
+const WARNING_HTML = `
+<details class="annoying-warning" open="">
+  <summary>This is a commit snapshot of the specification</summary>
+  <p>
+    This document contains the contents of the specification as of
+    <a href="https://github.com/tc39/ecma262/commit/${COMMIT}">commit ${COMMIT}</a>,
+    and should only be used as a historical reference. This commit may not
+    have even been merged into the specification.
+  </p>
+  <p>
+    Do not attempt to implement this version of the specification. Do not
+    reference this version as authoritative in any way. Instead, see
+    <a href="https://tc39.github.io/ecma262">https://tc39.github.io/ecma262</a>
+    for the living specification.
+  </p>
+</details>
+`;
+
+const WARNING_CSS = `
```

Comment:
could we store this in a .css file instead of inlining it?

Reply:
i think its better encapsulated as inlined, any reason to move it to a separate file?

Reply:
syntax highlighting, linting, reuse, etc